pom.xml

```diff
@@ -26,7 +26,7 @@
         <dependency>
             <groupId>io.github.java-diff-utils</groupId>
             <artifactId>java-diff-utils</artifactId>
-            <version>4.5</version>
+            <version>4.8-SNAPSHOT</version>
```

Comment:
Is it temporal `version`? Could you provide snapshot url or it is local build?

Reply:
How can I execute this branch on local?

Reply:
@timurt you can git clone https://github.com/java-diff-utils/java-diff-utils, and using mater branch, and run `mvn clean install ` to get `4.8-SNAPSHOT`